djstripe/models/billing.py

```diff
@@ -1830,6 +1825,85 @@ class SubscriptionSchedule(StripeModel):
         "values are `not_started`, `active`, `completed`, `released`, and "
         "`canceled`.",
     )
+    subscription = models.ForeignKey(
+        "Subscription",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="subscriptions",
+        help_text="ID of the subscription managed by the subscription schedule.",
+    )
+
+    def update(self, api_key=None, stripe_account=None, **kwargs):
+        """
+        Updates an existing subscription schedule
+        and returns the updated SubscriptionSchedule.
+
+        :param api_key: The api key to use for this request.
+            Defaults to djstripe_settings.STRIPE_SECRET_KEY.
+        :type api_key: string
+        :param stripe_account: The optional connected account \
+            for which this request is being made.
+        :type stripe_account: string
+        """
+        stripe_subscription_schedule = self._api_update(
+            api_key=api_key, stripe_account=stripe_account, **kwargs
+        )
+        return SubscriptionSchedule.sync_from_stripe_data(stripe_subscription_schedule)
+
+    def cancel(self, api_key=None, stripe_account=None, **kwargs):
+        """
+        Cancels a subscription schedule and its associated subscription immediately
+        (if the subscription schedule has an active subscription). A subscription schedule can only be canceled if its status is not_started or active
```

Comment:
Can you watch the line length please

Reply:
Looks like this is a bug in pre-commit because these things should have been caught. I'll see what's the issue.

Reply:
I'm guessing pre-commit doesn't look at string line length, especially docstrings.